exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -318,7 +321,8 @@ func (prwe *PrwExporter) execute(ctx context.Context, writeReq *prompb.WriteRequ
 	req.Header.Add("Content-Encoding", "snappy")
 	req.Header.Set("Content-Type", "application/x-protobuf")
 	req.Header.Set("X-Prometheus-Remote-Write-Version", "0.1.0")
-	req.Header.Set("User-Agent", "OpenTelemetry-Collector/"+version.Version)
+	FormatHeaderValues(req, prwe.startInfo.ExeName, prwe.startInfo.Version, prwe.startInfo.GitHash)
+	AddToUserAgent(req, "X-Prometheus-Remote-Write-Version/0.1.0")
```

Comment:
It seems that all of this is static data being set into the UA header.  Can the UA value be pre-computed in the initializer and simply set on the request here?

Reply:
Sure, I will compute the string in the initializer of the PRW Exporter and store it in the PrwExporter struct